src/my-app.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path="../bower_components/polymer-ts/polymer-ts.d.ts" />
```

Comment:
For now this will have to be the way, but this will get better with TypeScript 2.0 where you can configure where a module gets resolved to.